core/modules/file/file.pages.inc

```diff
@@ -559,6 +567,18 @@ function file_manage_form($form, &$form_state, File $file) {
     '#default_value' => file_uri_scheme($file->uri),
   );
 
+
+  if (!empty($file->uri)) {
+    $directory_path = file_stream_wrapper_get_instance_by_uri($file->uri)->getDirectoryPath();
+    $filename = file_uri_target($file->uri);
+    $filepath = $directory_path . '/' . $filename;
+    $form['destination']['url'] = array(
+      '#type' => 'item',
+      '#title' => t('File URL'),
+      '#markup' => url(check_plain($filepath), array('absolute' => TRUE)),
```

Comment:
Sorry, I know you just changed this, but I agree with @klonos's feedback in the issue that this should probably be a link, not just the URL as text.

![image](https://user-images.githubusercontent.com/100206/164949451-3fc148ae-9763-477a-816a-e544a0d0298f.png)

```suggestion
      '#markup' => l($filepath, $filepath, array('absolute' => TRUE)),
```

Though this could cause a new UX issue that clicking the link may loose changes on the form. We have an existing behavior in `Backdrop.behaviors.filePreviewLinks` that might help, but that might be out of scope?